src/app/components/modules/module-config.ts

```diff
@@ -9,7 +9,7 @@ import { SimplifiedScheduleComponent } from './simplified-schedule/simplified-sc
 import { SnowdayComponent } from './snowday/snowday.component';
 // import { StickynotesComponent } from './stickynotes/stickynotes.component';
 import { WeatherComponent } from './weather/weather.component';
-
+import { SimplifiedLunchComponent} from './simplified-lunch/simplified-lunch.component';
```

Comment:
I think the imports are arranged in alphabetical order. Could you also move the `SimplifiedLunchComponent` alphabetically in the `moduleComponents` array?